python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,318 @@
+
+        url = f"http://{self.location}/manager/api/get-token/"
+        data = {
+            "username": self.username,
+            "password": self.password,
+        }
+        resp = requests.post(url, data=data)
+        try:
+            token = resp.json()["token"]
+        except Exception:
+            raise Exception("Authentication failed")
+        self.websocket_url = (
+            f"ws://{self.location}/manager/ws/subscription?token={token}"
+        )
+
+    async def handle_message_reception(self):
+        """Handles the reception of messages."""
+
+        if self.websocket:
+            async for message in self.websocket:
+                if message.type == aiohttp.WSMsgType.TEXT:
+                    msg = json.loads(message.data)
+                    if "category" not in msg or (
+                        "option" in msg and msg["option"] == "subscribe"
+                    ):
+                        continue
+                    self.received_messages = self.received_messages + 1
+                    tracing = msg["tracing"]
+                    tracing["client_rcv"] = utils.current_tai()
+                    self.msg_traces.append(tracing)
+
+    async def subscribe_to(self, csc, salindex, stream, topic_type):
+        """Subscribes to the specified stream"""
+
+        subscribe_msg = {
+            "option": "subscribe",
+            "category": topic_type,
+            "csc": csc,
+            "salindex": salindex,
+            "stream": stream,
+        }
+        await self.websocket.send_str(json.dumps(subscribe_msg))
+
+    async def start_ws_client(self):
+        """Start client websocket connection"""
+
+        async with aiohttp.ClientSession() as session:
+            self.websocket = await session.ws_connect(self.websocket_url)
+            for name in self.event_streams:
+                csc, salindex = salobj.name_to_name_index(name)
+                for stream in self.event_streams[name]:
+                    await self.subscribe_to(csc, salindex, stream, "event")
+            for name in self.telemetry_streams:
+                csc, salindex = salobj.name_to_name_index(name)
+                for stream in self.telemetry_streams[name]:
+                    await self.subscribe_to(csc, salindex, stream, "telemetry")
+            await self.handle_message_reception()
+
+
+class StressLOVE(salobj.BaseScript):
+    """Run a stress test for one or more CSCs.
+
+    Notes
+    -----
+    **Details**
+
+    * Run a LOVE stress test by generating several client connections
+    that will listen to every event and telemetry of the specified CSCs
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Run a stress test for one or more CSCs")
+
+        # number of clients to simulate on the stress test
+        self.number_of_clients = 10
+
+        # number of messages to store
+        # before calculating the latency
+        self.number_of_messages = 1000
```

Comment:
It'd be nice to know that these are set by configuration (and you should add location, as well). But a simpler, and I think better, solution is to replace these with a `self.config` attribute, initialize it to `None` and then in the `configure` method set it to the provided configuration. Then access these values as `self.config.number_of_clients`, etc.

This simplifies the code, reduces duplication, and eliminates the confusion and hassle of assigning initial values here that never used (since they are overwritten by `configure`).